### PR TITLE
style:sidebar and layout style modify

### PR DESCRIPTION
**Pull request by tangjie-93**

### CODE REVIEW

File: themes/docsy/assets/scss/support/_utilities.scss [hunk 1]
```diff
@@ -76,7 +76,7 @@
 
 .td-max-width-on-larger-screens {
     @include media-breakpoint-up(lg) {
-        max-width: 80%;
+        max-width: 100%;
     }
 
 }
```
Comment (imbajin) [inline themes/docsy/assets/scss/support/_utilities.scss:82]: could add a empty line here

**Review by imbajin — COMMENTED** [review]
can we paste a picture to show the difference? thanks

**Comment by tangjie-93** [comment]
image

**Comment by tangjie-93** [comment]
**modify the height of sideBar** 
2d948cf847181562414af64256cddf31
d913b3dd9c11aa683a3d7db779e8e4ac

**modify the width of dropDown**
c7728dbb1aa61e4a4c70badf06a5c30d

**modify the width of content**

de257c972f22bf9a05c0085ceeafff21

**Comment by tangjie-93** [comment]
**modify the height of sideBar** 
2d948cf847181562414af64256cddf31
d913b3dd9c11aa683a3d7db779e8e4ac

**modify the width of dropDown**
c7728dbb1aa61e4a4c70badf06a5c30d
26e5a6b5f5d2b488f16f9f097723c148

**modify the width of content**
de257c972f22bf9a05c0085ceeafff21
bc9c5182862cf4da4f4c2c3859fb8300

**Pull request merged**